api/graphql/tests/test_application_events/base.py

```diff
@@ -143,3 +152,68 @@ def setUpTestData(cls) -> None:
         )
 
         cls.api_client = APIClient()
+
+
+class ApplicationEventPermissionsTestCaseBase(ApplicationEventTestCaseBase):
+    def create_service_sector_admin(self, service_sector=None):
+        service_sector_admin = get_user_model().objects.create(
+            username="ss_admin",
+            first_name="Amin",
+            last_name="Dee",
+            email="amin.dee@foo.com",
+        )
+
+        ServiceSectorRole.objects.create(
+            user=service_sector_admin,
+            role=ServiceSectorRoleChoice.objects.get(code="admin"),
+            service_sector=service_sector
+            or self.application.application_round.service_sector,
+        )
+        ServiceSectorRolePermission.objects.create(
+            role=ServiceSectorRoleChoice.objects.get(code="admin"),
+            permission="can_handle_applications",
+        )
+
+        return service_sector_admin
+
+    def create_unit_admin(self):
+        unit_group_admin = get_user_model().objects.create(
+            username="ss_admin",
+            first_name="Amin",
+            last_name="Dee",
+            email="amin.dee@foo.com",
+        )
+
+        unit_role = UnitRole.objects.create(
+            user=unit_group_admin,
+            role=UnitRoleChoice.objects.get(code="admin"),
+        )
+        UnitRolePermission.objects.create(
+            role=UnitRoleChoice.objects.get(code="admin"),
+            permission="can_validate_applications",
+        )
+
+        unit_role.unit.add(self.event_reservation_unit.reservation_unit.unit)
+
+        return unit_group_admin
+
+    def create_unit_group_admin(self):
+        unit_admin = get_user_model().objects.create(
+            username="ss_admin",
```

Comment:
Nitpick: if `ss_` stands for ServiceSector, perhaps it makes sense to use `ug_` here 😄